app/src/main/java/io/homeassistant/companion/android/widgets/template/TemplateWidget.kt

```diff
@@ -65,7 +65,7 @@ class TemplateWidget : BaseWidgetProvider() {
             if (widget != null) {
                 var renderedTemplate = templateWidgetDao.get(appWidgetId)?.lastUpdate ?: "Loading"
                 try {
-                    renderedTemplate = integrationUseCase.renderTemplate(widget.template, mapOf())
+                    renderedTemplate = integrationUseCase.renderTemplate(widget.template, mapOf()) ?: "Error"
```

Comment:
Hardcoded "Error" string, you should use a string resource (or even better, in case of `null` use the error view that is used for exceptions, to allow the latest successfully rendered template to remain visible).

Reply:
If we don't get a response then we should show the error screen.  Fixed.

Reply:
Resolved upstream already